### PR TITLE
Feature/confirm post deletion

### PR DESCRIPTION
**Pull request by lifenautjoe**
Added a confirm post and comment deletion bottom sheet. It's generic, so we can use it elsewhere.

Also made bottom sheets have rounded corners. It was annoying me for a long time already their squareness.

### CODE REVIEW

File: lib/pages/home/bottom_sheets/confirm_action.dart [hunk 1]
```diff
@@ -76,8 +76,7 @@ class OBConfirmActionBottomSheetState
     List<Widget> columnItems = [
       OBText(
         confirmationText,
-        size: OBTextSize.extraLarge,
-        style: TextStyle(fontWeight: FontWeight.bold),
+        style: TextStyle(fontWeight: FontWeight.bold, fontSize: 24),
```
Comment (Komposten) [inline lib/pages/home/bottom_sheets/confirm_action.dart]: Why did you change this from `OBTextSize.extraLarge` to the constant `24`?
(Not necessarily an issue, I'm just curious as to why.)

**Comment by duichwer** [comment]
I can't check the source from my phone, but have you also rearanged the actions of edit and delete that post and comments are ordered the same way?

File: lib/pages/home/bottom_sheets/confirm_action.dart (hunk 1 above)
Comment (lifenautjoe) [reply lib/pages/home/bottom_sheets/confirm_action.dart]: extra large is too large and large is too small 😢

**Comment by lifenautjoe** [comment]
@duichwer now it does!
